components/textfield/index.css

```diff
@@ -13,23 +13,24 @@ governing permissions and limitations under the License.
 @import '../commons/index.css';
 
 :root {
-  --spectrum-textfield-padding-top: 3px;
-  --spectrum-textfield-padding-bottom: 5px;
+  --spectrum-textfield-quiet-invalid-background-position: 100% 50%;
 
-  /* Todo fix in DNA */
-  --spectrum-textfield-quiet-border-radius: 0;
+  --spectrum-textarea-height: auto;
 
-  --spectrum-textfield-quiet-invalid-background-position: 100% 50%;
+  /* Todo: fix in DNA */
+  --spectrum-textarea-min-height: var(--spectrum-global-dimension-size-700);
 
-  --spectrum-textfield-multiline-height: auto;
-  --spectrum-textfield-multiline-min-height: var(--spectrum-global-dimension-size-700);
+  /* Remove border width from padding */
+  --spectrum-textfield-padding-top-adjusted: calc(var(--spectrum-textfield-padding-top) - var(--spectrum-textfield-border-size));
+  --spectrum-textfield-padding-bottom-adjusted: calc(var(--spectrum-textfield-padding-bottom) - var(--spectrum-textfield-border-size));
+  --spectrum-textfield-padding-x-adjusted: calc(var(--spectrum-textfield-padding-x) - var(--spectrum-textfield-border-size));
 
-  --spectrum-textfield-multiline-padding-x: var(--spectrum-textfield-padding-x);
-  --spectrum-textfield-multiline-padding-top: var(--spectrum-textfield-padding-top);
-  --spectrum-textfield-multiline-padding-bottom: var(--spectrum-textfield-padding-bottom);
+  --spectrum-textarea-padding-top-adjusted: calc(var(--spectrum-textarea-padding-top) - var(--spectrum-textarea-border-size));
+  --spectrum-textarea-padding-bottom-adjusted: calc(var(--spectrum-textarea-padding-bottom) - var(--spectrum-textarea-border-size));
+  --spectrum-textarea-padding-x-adjusted: calc(var(--spectrum-textarea-padding-x) - var(--spectrum-textarea-border-size));
```

Comment:
So this was very weird. If I overrode the same variable name, i.e.:

```css
--spectrum-textarea-padding-top: calc(var(--spectrum-textarea-padding-top) - var(--spectrum-textarea-border-size));
```

The output was:

```css
calc(
  var(--spectrum-textfield-padding-top, var(--spectrum-global-dimension-size-75))
  - var(--spectrum-textfield-border-size, var(--spectrum-alias-border-size-thin))
  - var(--spectrum-textfield-border-size, var(--spectrum-alias-border-size-thin))
)
```

But if I changed the name to `--spectrum-textarea-padding-top-adjusted`:

```css
--spectrum-textarea-padding-top-adjusted: calc(var(--spectrum-textarea-padding-top) - var(--spectrum-textarea-border-size));
```


I got the expected output:

```css
calc(
  var(--spectrum-textfield-padding-top, var(--spectrum-global-dimension-size-75))
  - var(--spectrum-textfield-border-size, var(--spectrum-alias-border-size-thin))
)
```

I'm not sure why this is happening, we should probably investigate further... It might be that the fallback variable substitution is happening incorrectly...

Reply:
Could you please create an issue to keep track of the further investigation for this weird issue ?